src/components/TextAreaInput/TextAreaInput.tsx

```diff
@@ -15,13 +14,14 @@ const TextAreaInput: React.FC<TextAreaInputProps> = ({ name, placeholder }) => {
   const errorMessage = errors[name]?.message as string | undefined;
 
   return (
-    <div className="text-area">
+    <div className="flex flex-col items-center">
       <textarea
-        className="text-area-input"
+        className="w-full h-[167px] rounded-[12px] p-3.5 bg-[#E9E8D5] mb-2.5 box-border 
+        text-base font-roboto leading-[19.2px] resize-none"
```

Comment:
Please discuss with Joshua the color and styling of his validation. 
On his pull request for the phone and email he used background color for the text input on an invalid input.